.github/DEVELOPMENT.md

```diff
@@ -0,0 +1,86 @@
+# Development
+
+After [forking the repo from GitHub](https://help.github.com/articles/fork-a-repo) and [installing pnpm](https://pnpm.io/installation):
```

Comment:
Do you want to add a quickstart on GitHub Codespaces too, so we have instructions both for local and remote development? 😁

Reply:
yes!